README.md

```diff
@@ -171,8 +177,82 @@ datalad containers-run \
         --output . \
         '{inputs}' '{outputs}' participant group -w workdir
 )
+
+```
+</details>
+
+## Walkthrough
+
+For users who are new to these components, we will walk through how
+these components are used together in a typical Yoda workflow.
+the steps
+
+```bash
+mkdir ~/my-experiments
+cd ~/my-experiments
+datalad create -d ds000003-qc -c text2git
+cd ds000003-qc
+```
+
+Datalad has created a new directory for our results, `ds000003-qc`.
+According to YODA principles, this dataset should also contain our input
+data, code, and anything else we need to run the analysis.
+
+Install the input dataset:
+
+```bash
+datalad install -d . -s https://github.com/ReproNim/ds000003-demo sourcedata
+```
+
+Now we install the `ReproNim/containers` collection.
+
+```bash
+datalad install -d . ///repronim/containers
 ```
 
+Now let's take a look at what we have.
```

Comment:
Try to adjust avoiding starting multiple sequentially appearing sentences with `Now`.

Although I don't mind such detailed walk through -- we need to add a comment somehow/somewhere that in case of change of that monolythic script (which is "smoke tested" by CI) - we would need to adjust these steps too.

Reply:
Perhaps a better place for the tutorial would be on readthedocs.

IIRC, the tutorial could render code directly from the script, so we remove duplication.

Reply:
"Now" fixed tho